routes/chat.js

```diff
@@ -49,6 +50,12 @@ router.post('/init-chat', auth.isAuth, chatController.initChat);
 router.post('/init-chat-anonymous', auth.isAuthAnonim, chatController.initChatAnonymous);
 router.post('/move-to-anon', auth.isAuthAnonim, validate(ChatValidation.moveToAnon), moveToAnon);
 router.post('/move-to-sign', auth.isAuthV2, validate(ChatValidation.moveToSign), moveToSign);
+router.post(
+  '/init-chat-from-post',
+  auth.isAuth,
+  validate(ChatValidation.initChatFromPost),
+  initChatFromPost
+);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of a new route `/init-chat-from-post` is correct. It's good to see that the route is protected by authentication middleware and uses validation middleware for request validation before passing control to the `initChatFromPost` function.
```diff
+ router.post(
+   '/init-chat-from-post',
+   auth.isAuth,
+   validate(ChatValidation.initChatFromPost),
+   initChatFromPost
+ );
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->